src/utils/users/users.test.js

```diff
@@ -0,0 +1,38 @@
+import * as serviceToken from '../token';
+import {getAnonymousUserId, getUserId} from './index';
+
+describe('users test should run correctly', () => {
+  // it('getUserId should run correctly', async () => {
+  //   const spyGetAccessToken = jest.spyOn(serviceToken, 'getAccessToken').mockResolvedValue({
+  //     id: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiMDE4OGJiYTMtNGRlZi00MDRiLTg3NmQtZWNkNTQyMTQ5Mzg5IiwiZXhwIjoxNjkyNDkyODY3fQ.2_iKOGQFbNNYjAmrd1Lnx0SChJxRs4fcmc2UsvJ4ars'
+  //   });
+
+  //   await getUserId();
+  //   expect(spyGetAccessToken).toHaveBeenCalled();
+  // });
+
+  // it('getAnonymousUserId should run correctly', async () => {
+  //   const spyGetAccessToken = jest
+  //     .spyOn(serviceToken, 'getAnonymousToken')
+  //     .mockResolvedValue(
+  //       'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiMDE4OGJiYTMtNGRlZi00MDRiLTg3NmQtZWNkNTQyMTQ5Mzg5IiwiZXhwIjoxNjkyNDkyODY3fQ.2_iKOGQFbNNYjAmrd1Lnx0SChJxRs4fcmc2UsvJ4ars'
+  //     );
+
+  //   await getAnonymousUserId();
+  //   expect(spyGetAccessToken).toHaveBeenCalled();
+  // });
+
+  it('getAnonymousUserId mock null should run correctly', async () => {
+    jest.spyOn(serviceToken, 'getAnonymousToken').mockResolvedValue(null);
+
+    const resp = await getAnonymousUserId();
+    expect(resp).toBeNull();
+  });
+
+  it('getAnonymousUserId catch error should run correctly', async () => {
+    jest.spyOn(serviceToken, 'getAnonymousToken').mockRejectedValue(null);
+
+    const resp = await getAnonymousUserId();
+    expect(resp).toBeNull();
+  });
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test cases for `getUserId` and `getAnonymousUserId` are commented out. If these tests are not needed, they should be removed. If they are needed, they should be uncommented and fixed if necessary.
```diff
-   // it('getUserId should run correctly', async () => {
-   //   const spyGetAccessToken = jest.spyOn(serviceToken, 'getAccessToken').mockResolvedValue({
-   //     id: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiMDE4OGJiYTMtNGRlZi00MDRiLTg3NmQtZWNkNTQyMTQ5Mzg5IiwiZXhwIjoxNjkyNDkyODY3fQ.2_iKOGQFbNNYjAmrd1Lnx0SChJxRs4fcmc2UsvJ4ars'
-   //   });
- 
-   //   await getUserId();
-   //   expect(spyGetAccessToken).toHaveBeenCalled();
-   // });
- 
-   // it('getAnonymousUserId should run correctly', async () => {
-   //   const spyGetAccessToken = jest
-   //     .spyOn(serviceToken, 'getAnonymousToken')
-   //     .mockResolvedValue(
-   //       'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiMDE4OGJiYTMtNGRlZi00MDRiLTg3NmQtZWNkNTQyMTQ5Mzg5IiwiZXhwIjoxNjkyNDkyODY3fQ.2_iKOGQFbNNYjAmrd1Lnx0SChJxRs4fcmc2UsvJ4ars'
-   //     );
- 
-   //   await getAnonymousUserId();
-   //   expect(spyGetAccessToken).toHaveBeenCalled();
-   // });
+   it('getUserId should run correctly', async () => {
+     const spyGetAccessToken = jest.spyOn(serviceToken, 'getAccessToken').mockResolvedValue({
+       id: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiMDE4OGJiYTMtNGRlZi00MDRiLTg3NmQtZWNkNTQyMTQ5Mzg5IiwiZXhwIjoxNjkyNDkyODY3fQ.2_iKOGQFbNNYjAmrd1Lnx0SChJxRs4fcmc2UsvJ4ars'
+     });
+
+     await getUserId();
+     expect(spyGetAccessToken).toHaveBeenCalled();
+   });
+
+   it('getAnonymousUserId should run correctly', async () => {
+     const spyGetAccessToken = jest
+       .spyOn(serviceToken, 'getAnonymousToken')
+       .mockResolvedValue(
+         'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiMDE4OGJiYTMtNGRlZi00MDRiLTg3NmQtZWNkNTQyMTQ5Mzg5IiwiZXhwIjoxNjkyNDkyODY3fQ.2_iKOGQFbNNYjAmrd1Lnx0SChJxRs4fcmc2UsvJ4ars'
+       );
+
+     await getAnonymousUserId();
+     expect(spyGetAccessToken).toHaveBeenCalled();
+   });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->